app/store/services/helpers/gps.js

```diff
@@ -60,9 +60,38 @@ const computeElevationGain = (points = []) =>
     return elevationGain;
   }, 0);
 
+const computeRegion = (locations = []) =>
```

Comment:
region should be defined by 2 points
I THINK THIS IS BETTER
const computeRegion = (...points)=>{
}

Reply:
how could it be defined only by two points?
-> a square (top, bottom, left, right) or a point and a radius!